core/java/iesi-core/src/main/java/io/metadew/iesi/connection/database/bigquery/DefaultBigqueryDatabaseConnection.java

```diff
@@ -0,0 +1,35 @@
+package io.metadew.iesi.connection.database.bigquery;
+
+import io.metadew.iesi.connection.database.bigquery.BigqueryDatabaseConnection;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.ToString;
+
+@Data
+@ToString(callSuper = true)
+@EqualsAndHashCode(callSuper = true)
+public class DefaultBigqueryDatabaseConnection extends BigqueryDatabaseConnection {
+
+    public DefaultBigqueryDatabaseConnection(String hostName, int portNumber, String project) {
+        super(getConnectionUrl(hostName, portNumber, project));
+    }
+
+    private static String getConnectionUrl(String hostName, int portNumber, String project) {
```

Comment:
why is this method static?

Reply:
Needs to be static since it is called before supertype constructor has been called

Reply:
@BerrevoetsRobbe  Have a look, but I think this could be kept as such